api/helpers/monitor.js

```diff
@@ -19,17 +20,24 @@ module.exports = app => {
       var topNav = $("#top_nav");
       var topNavText = topNav.text();
 
+      //checks if alternate parking banner exists
       if (topNavText.includes("Contact Us")) {
-        // gets the date the scrape is performed
-        // final implentation will get date from website scrape
-        let date = new Date();
+        var date = new Date();
+        //var offset = date.getTimezoneOffset() / 60;
+        // console.log(offset);
+        // date.setHours(date.getHours() - offset);
+        // console.log(date);
 
         // creates new status and sets attributes
         let newStatus = {
           alternateParking: true,
```

Comment:
Is there a way to make the `alternateParking` false?

Reply:
I guess there isn't. I didn't even think about that part. The way that it works now is that it just puts a new record in the database with an expiration date. But I guess if the banner isn't there anymore I could just update the database with a record that says it's not in effect any longer.

Reply:
Yeah I guess it's not necessary, but without it it isn't very explicit whether it's active or not. It's up to you how you want to implement it!